### ISSUE TITLE
Intake screen: calendar fields sometimes don't show the date 

### ISSUE THREAD

**Issue by uli-on** [pewaru-333/HomeMedkit-App#2]
~~On pressing the calendar buttons on the Intake screen nothing happens.~~

EDIT
The real issue is another one: https://github.com/pewaru-333/HomeMedkit-App/issues/2#issuecomment-1826410680

**Comment by pewaru-333** [comment]
Thank you. For now it just show the intake period.

**Comment by uli-on** [comment]
For me, with my screen resolution, they just show the generic calendar 
icons, no values.
( 1080x2016, density 3x)

**Comment by pewaru-333** [comment]
Thank you. I'll see what can cause this problem.

**Comment by pewaru-333** [comment]
> This is not caused by my 107% font size, BTW. I just tested the smallest size available, hoping the invisible dates would contain wider numerals, but alas, no.
> 
> But I discovered the two terms Start and Finish below the calendar fields, which I never saw before, because this page can not be scrolled. Shall I open a new topic for this issue?

Thank you. No, I understood the issue.

**Comment by uli-on** [comment]
The scrollability of the page is fixed, like quoted above. The issue that the calendar fields sometimes not shows the date still exists. Sorry.

**Comment by pewaru-333** [comment]
> The scrollability of the page is fixed, like quoted above. The issue that the calendar fields sometimes not shows the date still exists. Sorry.

Thank you. I will check it later.

**Comment by uli-on** [comment]
OK, so I experimented a little with the medicine in question and saw that it is only faulty when I use "other" for the period field, and only when I leave the medicine unsaved with that "other" value (created in a previous. Medkit version and unfortunately not shown on the edit screen). Any period else, any value I tried for "other" in the current app version produced a _visual_ date field. Hence I'd see this as fixed now, and will close the issue. Thanks :)
